src/Spd.Presentation.Licensing/Controllers/BizPortalUserController.cs

```diff
@@ -45,6 +48,30 @@ public async Task<BizPortalUserResponse> Add([FromBody][Required] BizPortalUserC
         return await _mediator.Send(new BizPortalUserCreateCommand(bizPortalUserCreateRequest, hostUrl, userId));
     }
 
+    /// <summary>
+    /// Update Business Portal User
+    /// </summary>
+    /// <param name="bizId"></param>
+    /// <param name="userId"></param>
+    /// <param name="bizPortalUserUpdateRequest"></param>
+    /// <returns></returns>
+    [Authorize(Policy = "OnlyBCeID", Roles = "PrimaryManager,Manager")]
+    [Route("api/business/{bizId}/portal-users/{userId}")]
+    [HttpPut]
+    public async Task<BizPortalUserResponse> Put([FromRoute] Guid bizId, [FromRoute] Guid userId, [FromBody][Required] BizPortalUserUpdateRequest bizPortalUserUpdateRequest)
+    {
+        bizPortalUserUpdateRequest.Id = userId;
+        bizPortalUserUpdateRequest.BizId = bizId;
+
+        //if role is manager, can only change his own information
+        if (_currentUser.GetUserRole() == ContactAuthorizationTypeCode.BusinessManager.ToString() &&
+            userId.ToString() != _currentUser.GetUserId())
+        {
+            throw new ApiException(HttpStatusCode.Forbidden, "Authorized Contact can only change his own information.");
```

Comment:
Should this say 'Business Manager' instead of 'Authorized Contact' ?

Reply:
nice catch. Please fix

Reply:
Nice catch, fixed